tests/unit_tests/test_array_contains.py

```diff
@@ -16,12 +16,14 @@ def array_contains_process_code():
         ({"data": [1, 2, 3], "value": 2}, True),
         ({"data": ["A", "B", "C"], "value": "b"}, False),
         ({"data": [1, 2, 3], "value": "2"}, False),
-        ({"data": [1, 2, None], "value": None}, False),
+        ({"data": [1, 2, None], "value": None}, False), # example in docs shows "True" as a return, but description states "False"
```

Comment:
Opened an issue for this: https://github.com/Open-EO/openeo-processes/issues/348

Feel free to open them yourself.